llm/gguf.go

```diff
@@ -79,6 +57,28 @@ func newGGUFModel(container *containerGGUF) *ggufModel {
 	}
 }
 
+func (c *containerGGUF) Name() string {
```

Comment:
nit: if this code hasn't changed, best to move it back up so the diff is easier to read